README.md

```diff
@@ -7,13 +7,26 @@ the following:
 
 1. Add markdown content to `pages/`
 2. Update `site-config.yml`
-    i. `hub` is the github slug for your active hub. This example defaults to the CDC FluSight hub
+    i. `hub` is the github slug for your active hub. This example defaults to
```

Comment:
In all honesty some folks might not be familiar with the term `slug` (it's very web devy). Should we be more explicit? ie. would GitHub repository `owner/name` be better?

Reply:
Good point! It's definitely jargon. 

```suggestion
    i. `hub` is the github repository (`owner/repo`) for your active hub. This example defaults to
```